src/tests/system/tests/test_logging.py

```diff
@@ -45,38 +50,91 @@ def test_logging__default_settings_logs_debug_level(client: Client):
 @pytest.mark.integration
 @pytest.mark.importance("low")
 @pytest.mark.topology(KnownTopology.Client)
-def test_logging__default_settings_does_not_log_user_logins(client: Client):
+def test_logging__custom_settings_logs_debug_level(client: Client):
+    """
+    :title: Domain log level from configuration is honored
+    :setup:
+        1. Configure SSSD for local system authentication
+        2. Clear logs and start SSSD with debug level 9
+    :steps:
+        1. Check log files
+    :expectedresults:
+        1. Logs messages contain configured debug level 0x4000
+    :customerscenario: False
+    """
+    client.sssd.common.local()
+    client.sssd.default_domain = "local"
```

Comment:
I think this is silly, I created https://github.com/SSSD/sssd-test-framework/pull/162/files so it's a part of sssd.common.local() , added you as a reviewer.